src/main/scala/rocket/ICache.scala

```diff
@@ -182,7 +182,8 @@ class ICacheModule(outer: ICache) extends LazyModuleImp(outer)
     name = "tag_array",
     desc = "ICache Tag Array",
     size = nSets,
-    data = Vec(nWays, UInt(width = tECC.width(1 + tagBits)))
+    data = Vec(nWays, UInt(width = tECC.width(1 + tagBits))),
+    id = DescribedSRAMIdAssigner.genId()
```

Comment:
Why does this get passed in instead of just getting called inside `DescribedSRAM.apply`?

Reply:
fixed